servicex_app/servicex_app/resources/servicex_resource.py

```diff
@@ -92,6 +93,9 @@ def start_transformers(cls, transformer_manager: TransformerManager,
         transformer_manager.launch_transformer_jobs(
             image=request_rec.image, request_id=request_rec.request_id,
             workers=request_rec.workers,
+            max_workers=(max(1, request_rec.files)
+                         if request_rec.status == TransformStatus.running
```

Comment:
Shouldn't this be limited to `TRANSFORMER_MAX_REPLICAS` in case the number of files is > than that number?

Reply:
Limit is applied in the autoscaler invocation https://github.com/ssl-hep/ServiceX/pull/897/files#diff-ab4dc02ce708be56b4ebde72961b66d0fabf04bc255fdb60154f88889711edc5R333

Reply:
I _think_ this point is addressed? (the max limit is applied during the HPA definition.) Let me know if not.